fjord/analytics/analyzer_views.py

```diff
@@ -166,12 +167,12 @@ def _analytics_search_export(request, opinions_s):
     keys = Response.get_export_keys(confidential=True)
     total_opinions = opinions_s.count()
 
-    opinions_s = opinions_s.values_list('id')[:MAX_OPINIONS]
+    opinions_s = opinions_s.fields('id')[:MAX_OPINIONS].execute()
```

Comment:
The ElasticUtils `values_list` and the elasticsearch-dsl `fields` essentially do the same thing.

With elasticsearch-dsl, you have to explicitly call `execute` to run the search and get back a `Results` instance.